### DISCUSSION TITLE
cats-effect 3 Release Plan

### DISCUSSION THREAD

**Discussion by juanpedromoreno** [planet42/Laika#238] [Q&A] [answered]
Hi 👋 

Thanks for the library! I can see in the README.md file 

> Support for Scala 3 and cats-effect 3 is available on master, but has not been released yet.

I wonder what are the plans for making a new release? Thanks in advance.

**Comment by jenshalm** [comment, marked as answer]
Hi there, the release is ready, but blocked on http4s doing a final release for cats-effect 3 and Scala 3, which is still pending. Once they are ready I will release within a few days.

**Comment by jenshalm** [comment]
http4s released on Friday, and Laika 0.18 for Scala 3 and cats-effect 3 is out today: https://github.com/planet42/Laika/releases/tag/0.18.0.

I'm closing this ticket now.

**Comment by juanpedromoreno** [comment]
Thanks @jenshalm !
